test/blackbox-tests/test-cases/dune-project-meta/basic-generate.t

```diff
@@ -29,12 +29,28 @@ The `dune build` should generate the opam file
   > (package
   >   (name cohttp-async)
   >   (synopsis "HTTP client and server for the Async library")
-  >   (description "A _really_ long description")
+  >   (description "\
+  > A _really_long multi line description that spans across lines to
+  > make sure that the rendering of long strings stays compatible.
+  > ")
   >   (depends
   >     (cohttp (>= 1.0.2))
   >     (conduit-async (>= 1.0.3))
   >     (async (>= v0.10.0))
   >     (async (< v0.12))))
+  > ;
```

Comment:
nitpick but is this comment necessary?

Reply:
No. The other lines had these comments so for consistency I added it. But it is not required so to be consistent I removed it from all lines.